installing/installing_aws/preparing-to-install-on-aws.adoc

```diff
@@ -26,6 +26,11 @@ You can install {product-title} on installer-provisioned or user-provisioned inf
 
 See xref:../../architecture/architecture-installation.adoc#installation-process_architecture-installation[Installation process] for more information about installer-provisioned and user-provisioned installation processes.
 
+[id="choosing-an-method-to-install-ocp-on-aws-single-node"]
```

Comment:
@johnwilkins the SNO is using IPI method, so personally I don't think we need to add SNO part in this section.

Reply:
@johnwilkins Do you have any comments or questions about my above comment?

Reply:
That is there for discovery purposes so people will find the SNO docs if they are deploying on AWS and looking in the AWS documentation.